.circleci/config.yml

```diff
@@ -372,15 +372,15 @@ jobs:
           name: Check Scripts
           command: /root/project/bin/check_shell_scripts.sh
 
-  lint:
-    <<: *defaults
-    environment:
-      KUBECONFIG: /go/src/istio.io/istio/.circleci/config
-    resource_class: xlarge
-    steps:
-      - checkout
-      - run: make sync
-      - run: make lint
+#  lint:
```

Comment:
I think we generally remove code rather than commenting it out. We have the git history anyways.

Reply:
Agree, I was following the approach taken by @rshriram when he disabled the other tests. I can delete all the commented out though if you prefer

Reply:
agreed please remove - we can revert if needed.

Reply:
sgtm, removed them all